worker/runtime/spec/spec.go

```diff
@@ -171,6 +174,18 @@ func OciResources(limits garden.Limits) *specs.LinuxResources {
 	}
 }
 
+var IsSwapLimitEnabled bool
+
+func init() {
+	IsSwapLimitEnabled = swapLimitEnabled()
+}
+
+func swapLimitEnabled() bool {
+	swapLimitFile := "/sys/fs/cgroup/memory/memory.memsw.limit_in_bytes"
```

Comment:
Can you move this to the top level var declaration? I like the pattern of defining any hard coded values at the top of the file.